src/app/onboarding/components/onboarding/onboarding.component.html

```diff
@@ -4,11 +4,14 @@
   </div>
 </div>
 <pr-debugger [currentScreen]="screen" (setState)="setState($event)"></pr-debugger>
-<div class="onboarding">
+<div class="onboarding" *ngIf="!skipOnboarding ; else loading">
```

Comment:
I find that positively-worded boolean variable names are much easier to read and reason about than negatively-worded ones, particularly when we then need to negate it to use! So maybe: `showOnboarding` instead?